rdmd_test.d

```diff
@@ -310,6 +310,16 @@ void runTests()
         assert(res.status == -SIGSEGV, format("%s", res));
     }
 
+    /* https://issues.dlang.org/show_bug.cgi?id=11997- rdmd should search it's
+    binary path for the compiler */
+    string localDMD = buildPath(tempDir(), baseName(compiler));
+    std.file.write(localDMD, "empty shell");
+    scope(exit) std.file.remove(localDMD);
+
+    res = execute(rdmdApp ~ [modelSwitch, "--force", "--chatty", "--eval=writeln(`Compiler found.`);"]);
+    assert(res.status == 1, res.output);
+    assert(res.output.canFind(`spawn ["` ~ localDMD ~ `",`));
```

Comment:
>I think you could do a "positive" test by putting a fake compiler binary in the same directory as rdmd, but it's entirely up to you.

You mean positive in that `rdmd` would run and exit successfully?  Wouldn't dmd need druntime and phobos and a config file too?  Becomes too much of a PITA, especially once you consider other compilers, like ldc.
 
>Another way would be looking at the output of --dry-run, and simply checking that it would have executed the correct compiler.

Yeah, that might work, but you're not even executing the `eval=writeln` code then, so not much of a difference.